app_dart/lib/src/request_handlers/reset_prod_task.dart

```diff
@@ -40,38 +39,18 @@ class ResetProdTask extends ApiRequestHandler<Body> {
   final Scheduler scheduler;
 
   static const String taskKeyParam = 'Key';
-  static const String ownerParam = 'Owner';
-  static const String repoParam = 'Repo';
-  static const String commitShaParam = 'Commit';
-  static const String builderParam = 'Builder';
-  static const String propertiesParam = 'Properties';
 
   @override
   Future<Body> post() async {
     final DatastoreService datastore = datastoreProvider(config.db);
     final String encodedKey = requestData![taskKeyParam] as String? ?? '';
     final KeyHelper keyHelper = config.keyHelper;
-    final String owner = requestData![ownerParam] as String? ?? 'flutter';
```

Comment:
This API is used by the engine team to trigger builds manually. Before landing this change we need to sync up with them and send proper PSAs

Reply:
I re-added this logic back, and I've synced up with the engine team to work on getting them to use the dashboard more